validator/rpc/wallet_test.go

```diff
@@ -34,6 +35,9 @@ func TestServer_CreateWallet_Direct(t *testing.T) {
 		WalletPassword:    strongPass,
 		KeystoresPassword: strongPass,
 	}
+	// We delete the directory at defaultWalletPath as CreateWallet will return an error if it tries to create a wallet
+	// where a directory already exists
+	require.NoError(t, os.RemoveAll(defaultWalletPath))
```

Comment:
This and also line 87 delete the directory created earlier in the test function by setupWalletDir().  If I don't then the check for an already existing wallet will return an error (the expected behavior in normal usage).  Unfortunately setupWalletDir is used by 11 tests so I thought it better to just leave it like I have it (just 2 changes) instead of changing that function and changing a bunch of tests to compensate.